### PR TITLE
cluster: add scaleup,down,recovering,upgrading conditions

### PR DESCRIPTION
**Pull request by xiang90**

### CODE REVIEW

File: pkg/cluster/reconcile.go [hunk 1]
```diff
@@ -162,6 +156,8 @@ func (c *Cluster) resize() error {
 }
 
 func (c *Cluster) addOneMember() error {
+	c.status.AppendScalingUpCondition()
```
Comment (hongchaodeng) [inline pkg/cluster/reconcile.go]: We also have self hosted path.

Comment (xiang90) [inline pkg/cluster/reconcile.go]: what do you mean? can you provide more details?

Comment (hongchaodeng) [inline pkg/cluster/reconcile.go]: If we call AppendScalingUpCondition() in addOneMember(), we also need to do the same in addOneSelfHostedMember().

Comment (xiang90) [inline pkg/cluster/reconcile.go]: right. done.

**Comment by hongchaodeng** [comment]
@etcd-bot retest this please

**Comment by xiang90** [comment]
@hongchaodeng All fixed. PTAL.

**Comment by hongchaodeng** [comment]
lgtm

**Pull request merged**